homeassistant/components/ads/binary_sensor.py

```diff
@@ -1,37 +1,29 @@
 """Support for ADS binary sensors."""
-import voluptuous as vol
-
 from homeassistant.components.binary_sensor import (
     DEVICE_CLASS_MOVING,
-    DEVICE_CLASSES_SCHEMA,
-    PLATFORM_SCHEMA,
     BinarySensorEntity,
 )
-from homeassistant.const import CONF_DEVICE_CLASS, CONF_NAME
-import homeassistant.helpers.config_validation as cv
+from homeassistant.const import CONF_BINARY_SENSORS, CONF_DEVICE_CLASS, CONF_NAME
 
 from . import CONF_ADS_VAR, DATA_ADS, STATE_KEY_STATE, AdsEntity
 
-DEFAULT_NAME = "ADS binary sensor"
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {
-        vol.Required(CONF_ADS_VAR): cv.string,
-        vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_DEVICE_CLASS): DEVICE_CLASSES_SCHEMA,
-    }
-)
-
 
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the Binary Sensor platform for ADS."""
+    entities = []
+
+    if discovery_info is None:  # pragma: no cover
```

Comment:
Why `# pragma: no cover` when there are not tests currently?
```suggestion
    if discovery_info is None:
```